.github/workflows/CI.yml

```
@@ -45,5 +45,8 @@ jobs:
     - name: 5. Grant execute permission for gradlew
       run: chmod +x gradlew
 
-    - name: 6. Build with Gradle
-      run: ./gradlew clean build
+    - name: 6. Redis Server in GitHub Actions
+      uses: supercharge/redis-github-action@1.8.0
+
+    - name: 7. Build with Gradle
```

actionlint (1.7.4)

step must run script with "run" section or run action with "uses" section

(syntax-check)